### PR TITLE
London class 7 - Tanika Robinson - HTML/CSS - WEEK 2

### PR DESCRIPTION
**Pull request by tanikaDev**
Edited the index and css files to closely resemble desktop and mobile webpages provided.

# Your Details

- Your Name: Tanika
- Your City: London
- Your Slack Name: Tanika 

# Homework Details

- Module: HTML/CSS
- Week: 2

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,11 +1,199 @@
...
+    width: 30px;
+    height: 30px;
+}
+
+i::before {
+    position: absolute;
+    margin: 6px 0 0 -6px;
+}
+
+/* MEDIA QUERIES */
```
Comment (Behnazz) [inline css/style.css:177]: 👏

File: css/style.css [hunk 2]
```diff
@@ -1,11 +1,199 @@
...
+
+i {
+    position: relative;
+    border: 1px solid rgb(192, 192, 192);
+    border-radius: 30px;
+    width: 30px;
+    height: 30px;
+}
+
+i::before {
```
Comment (Behnazz) [inline css/style.css:172]: 👍

**Review by Behnazz — COMMENTED** [review]
WellDone! Your work was very good. Just some minor comments here or there.

File: css/style.css [hunk 3]
```diff
@@ -1,11 +1,199 @@
...
 
 body {
     font-family: 'Roboto', sans-serif;
+    font-weight: 300;
     -webkit-font-smoothing: antialiased;
 }
 
+h1, h2 {
+    font-family: 'Roboto', sans-serif;
+    font-weight: 300;
```
[the remark this answers is not in the document]
Comment (Trobinsonshoots) [reply css/style.css:17]: thanks Behnaz, I fully agree with that. The only thing is when I was writing my code it wasn't applying the font to the heading tags for some weird reason so did that to get around it. i will try take it out again and see if it works

File: css/style.css [hunk 4]
```diff
@@ -1,11 +1,199 @@
...
+    margin: 0;
+}
+
+p {
+
+}
```
[the remark this answers is not in the document]
Comment (Trobinsonshoots) [reply css/style.css:23]: lol forgot to take it out, will do now.

File: css/style.css [hunk 5]
```diff
@@ -1,11 +1,199 @@
...
+
+/* HEADER */
+
+/* NAV */
```
[the remark this answers is not in the document]
Comment (Trobinsonshoots) [reply css/style.css:27]: I did that as everything below is in the header and the first css edits were for the nav, so nav followed with the edits underneath. I was suppose to help with readability if anyone is going in but I guess it's confusing with them both together. Will take header out

**Pull request closed without merging**